com.unity.multiplayer.mlapi/Runtime/Core/NetworkObject.cs

```diff
@@ -214,18 +216,19 @@ public void NetworkShow(ulong clientId, Stream payload = null)
         /// <param name="payload">An optional payload to send as part of the spawns</param>
         public static void NetworkShow(List<NetworkObject> networkObjects, ulong clientId, Stream payload = null)
         {
-            if (!NetworkManager.Singleton.IsServer)
-            {
-                throw new NotServerException("Only server can change visibility");
-            }
-
             if (networkObjects == null || networkObjects.Count == 0)
             {
                 throw new ArgumentNullException("At least one NetworkObject has to be provided");
             }
 
             NetworkManager networkManager = networkObjects[0].NetworkManager;
```

Comment:
Since we now have a NetworkManager per NetworkObject, can we not just read its NM here vs. assume the 0th one?

Reply:
Read comments on other related review.